frontend/app/src/App.test.tsx

```diff
@@ -774,6 +775,30 @@ describe("App.handleNewSession", () => {
       // @ts-expect-error
       window.history.pushState.mockClear()
     })
+
+    it("logs a warning when hostConfig.disableSetPageMetadata is true", async () => {
+      const ref = React.createRef()
```

Comment:
Why do we need the `const ref = React.createRef()` here? Can't we just remove it since it's unused?

Reply:
Oops yes. I thought I was using it in the reference when rendering app but that's wrong.

Reply:
Done. Tests should run and pass now... thanks.